test-engineering/contract-tests/client/tests/models.py

```diff
@@ -21,6 +21,8 @@ class Request(BaseModel):
     method: str
     path: str
     headers: List[Header] = []
+    # Delay is optional, providing time for data refresh
+    delay: Optional[int] = 0
```

Comment:
Should this be a float?

I just saw that the 'sleep' function supports it, so perhaps we should propagate the precision?
<img width="570" alt="image" src="https://user-images.githubusercontent.com/3422688/183693468-3b4a83ea-1d09-4200-b5f8-28934a99b46e.png">

Reply:
Good point, will change to a float!